apps/e2e/cypress/e2e/back-office-applications/documentDownload.spec.js

```diff
@@ -29,7 +29,7 @@ describe('Document Download', () => {
 	});
 
 	it('Case Team Admin user should be able to download a document', () => {
-		cy.login(applicationsUsers.caseAdmin);
+		const fileName = 'sample-doc.pdf';
```

Comment:
Do we not need the login steps now?

Reply:
this is handled in the `before` hook and the user stays signed in